hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/clientrm/TestFederationClientInterceptorRetry.java

```diff
@@ -160,19 +165,13 @@ protected YarnConfiguration createConfiguration() {
    */
   @Test
   public void testGetNewApplicationOneBadSC()
```

Comment:
Single line

Reply:
Thanks for your suggestion, I will fix it.